backend/tests/init_data/skills/sandbox/test_download_attachment_tool.py

```
@@ -0,0 +1,96 @@
+
+    monkeypatch.setattr(
+        SandboxDownloadAttachmentTool,
+        "_get_sandbox_manager",
+        lambda self: FakeManager(),
+    )
+    tool = SandboxDownloadAttachmentTool(
+        task_id=1,
+        subtask_id=2,
+        user_id=3,
+        user_name="alice",
+        auth_token="task-token",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)